lib/models/project.js

```diff
@@ -216,6 +216,39 @@ class Project {
   }
 
   /**
+    Returns the targets of this project, or the default targets if not present
+
+    @public
+    @method targets
+    @return {Object}  Targets object
+   */
+  get targets() {
+    let configPath = 'config';
+
+    if (this.pkg['ember-addon'] && this.pkg['ember-addon']['configPath']) {
+      configPath = this.pkg['ember-addon']['configPath'];
+    }
+
+    let targetsPath = path.join(this.root, configPath, 'targets');
+
+    let appTargets;
+    if (existsSync(`${targetsPath}.js`)) {
+      appTargets = JSON.parse(fs.readFileSync(`${targetsPath}.js`, 'utf8'));
```

Comment:
I believe this should use `require` since the targets file isn't valid JSON and we want to allow usage of node-land things anyways.